lib/model/content.dart

```diff
@@ -351,26 +351,49 @@ class ImageNodeList extends BlockContentNode {
 }
 
 class ImageNode extends BlockContentNode {
-  const ImageNode({super.debugHtmlNode, required this.srcUrl});
+  const ImageNode({
+    super.debugHtmlNode,
+    required this.srcUrl,
+    required this.thumbnailUrl,
+    required this.loading,
+  });
 
-  /// The unmodified `src` attribute for the image.
+  /// The canonical source URL of the image.
   ///
-  /// This may be a relative URL string.  It also may not work without adding
+  /// This may be a relative URL string. It also may not work without adding
   /// authentication credentials to the request.
   final String srcUrl;
 
+  /// The thumbnail URL of the image.
+  ///
+  /// This may be a relative URL string. It also may not work without adding
+  /// authentication credentials to the request.
+  /// It may be null if Server didn't generate a thumbnail, or doesn't support
+  /// it yet. It will also be null when [loading] is true.
```

Comment:
```suggestion
  ///
  /// This will be null if the server hasn't yet generated a thumbnail,
  /// or is a version that doesn't offer thumbnails.
  /// It will also be null when [loading] is true.
```

(in particular I found the meaning of "yet" a bit hard to parse in the current revision)